cassandra/cluster.py

```diff
@@ -372,6 +373,14 @@ def auth_provider(self, value):
     If set to :const:`None`, there will be no timeout for these queries.
     """
 
+    idle_heartbeat_interval = 30
```

Comment:
I understand the desire to make the CC more responsive. It would actually be pretty easy to refactor and put a dedicated thread with its own interval on CC. That would be the cleanest, design-wise. I didn't do that in this iteration because I wasn't convinced it was worth the overhead.
Another option would be to run the loop at a higher frequency and only hit connections with is_control_connection set.
Since it's a late in the dev cycle I might propose this as a follow-on improvement.